src/invidious/comments.cr

```diff
@@ -674,6 +689,18 @@ def content_to_comment_html(content, video_id : String? = "")
     text = "<b>#{text}</b>" if run["bold"]?
     text = "<s>#{text}</s>" if run["strikethrough"]?
     text = "<i>#{text}</i>" if run["italics"]?
+    if run["emoji"]?
+      if run["emoji"]["isCustomEmoji"]?.try &.as_bool
+        if emojiImage = run.dig?("emoji", "image")
+          emojiAlt = emojiImage.dig?("accessibility", "accessibilityData", "label").try &.as_s || text
+          emojiThumb = emojiImage["thumbnails"][0]
+          text = "<img alt=\"#{emojiAlt}\" src=\"/ggpht#{URI.parse(emojiThumb["url"].as_s).request_target}\" title=\"#{emojiAlt}\" width=\"#{emojiThumb["width"]}\" height=\"#{emojiThumb["height"]}\" style=\"margin-right:2px;margin-left:2px;\"/>"
```

Comment:
For clarity, please use either `String.build` or [Heredocs] (`<<-HTML`).
If you're using `String.build`, you can use [percent string literals](https://crystal-lang.org/reference/1.6/syntax_and_semantics/literals/string.html#percent-string-literals) `%()` to write strings with a lot of quotes. E.g:
```crystal
output_string = String.build do |str|
  str << %(<img alt=") << emojiAlt << '"'
  # etc...
  str << %( style="margin-right:2px;margin-left:2px;"/>)
end
```

Reply:
Done 🙂